### DISCUSSION TITLE
Official SNES controller for Switch - home button

### DISCUSSION THREAD

**Discussion by Panja0** [ndeadly/MissionControl#740] [Q&A]
Simple question but I can't seem to find a straight answer.

When using the official Nintendo SNES controller for Switch can I use a(ny) button combo to use as the home button?

**Comment by ndeadly** [comment]
Mission Control hardcodes button combos for capture and home buttons for controllers that don't support them. However, Nintendo does input masking on Joy-Cons and possibly other official controllers to restrict the inputs to only those present on the controller.

Try it and see if it works. The combo for Home is `MINUS` (I guess that's `SELECT` on the SNES controller) + `DPAD_DOWN`. If it doesn't, then the inputs are masked for the SNES controller. I'm yet to find a workaround for this.

**Reply by Panja0** [reply]
Thanks for the ultra fast answer!
About testing....

I wanted to know if the home button is functional so I can actually buy a SNES controller. :)
I don't have one right now and not having the home button is a showstopper for me. Thence my question.

**Reply by ndeadly** [reply]
Yeah, I can't really answer that as I don't own one. I'll ask on my discord server if anyone can give me an answer.

**Reply by Panja0** [reply]
Thanks, I understand! Appreciate it, thanks for helping out.
